cmndtoolsbraker/ls.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+import os
+import sys
+
+if len(sys.argv) == 1:
+    files = []
+    for f in os.listdir('.'):
+        if not f.startswith('.'):
+            files.append(f)
+    print(' '.join(files))
+elif len(sys.argv) == 2:
+    if sys.argv[1] == '-a' or sys.argv[1] == '-all':
+        print(' '.join(os.listdir('.')))
```

Comment:
```suggestion
        print('\n'.join(os.listdir('.')))
```
Удобнее разделить вывод на разные строки. Такой вывод удобнее потом передавать дальше по конвейеру